pdns/recursordist/docs/running.rst

```diff
@@ -111,4 +111,97 @@ Do not forget to disable tracing after diagnosis is done::
+| **Key**               | **Description**             | **Remarks**                         |
++-----------------------+-----------------------------+-------------------------------------+
+|``ecs``                |Client ECS info              |Filled in if enabled                 |
++-----------------------+-----------------------------+-------------------------------------+
+|``proto``              |Protocol used by client      |``udp`` or ``tcp``                   |
++-----------------------+-----------------------------+-------------------------------------+
+|``qname``              |Query name                   |                                     |
++-----------------------+-----------------------------+-------------------------------------+
+|``qtype``              |Query type                   |                                     |
++-----------------------+-----------------------------+-------------------------------------+
+|``remote``             |Client address               |IP:port combination                  |
++-----------------------+-----------------------------+-------------------------------------+
+|                               **Keys specific to Answers**                                |
++-----------------------+-----------------------------+-------------------------------------+
+|``additional``         |Number of additional records |                                     |
+|                       |in answer                    |                                     |
++-----------------------+-----------------------------+-------------------------------------+
+|``answer-is-variable`` |Is answer marked variable by |e.g. ECS dependent answers           |
+|                       |recursor?                    |                                     |
++-----------------------+-----------------------------+-------------------------------------+
+|``answers``            |Number of answer records in  |                                     |
+|                       |answer                       |                                     |
++-----------------------+-----------------------------+-------------------------------------+
+|``dotout``             |Number of outgoing DoT       |                                     |
+|                       |queries sent to authoritative|                                     |
+|                       |servers to resolve answer    |                                     |
++-----------------------+-----------------------------+-------------------------------------+
+|``into-packetcache``   |Is the answer being stored   |Variable answers (as determined by   |
+|                       |into the packetcache?        |the recursor) or answers marked as   |
+|                       |                             |variable by Lua code will not be put |
+|                       |                             |into the packet cache.               |
```

Comment:
Perhaps `Variable answers (as determined by the recursor or marked as such by Lua code) will not be put into the packet cache`?